pwndbg/chain.py

```diff
@@ -44,8 +48,14 @@ def get(address, limit=LIMIT, offset=0):
 config_arrow_right = theme.Parameter('chain-arrow-right', '—▸', 'right arrow of chain formatting')
 config_contiguous  = theme.Parameter('chain-contiguous-marker', '...', 'contiguous marker of chain formatting')
 
-def format(value, limit=LIMIT, code=True, offset=0):
-    chain = get(value, limit, offset)
+def format(value, limit=LIMIT, code=True, offset=0, hard_stop=None, hard_end=0):
```

Comment:
Added a docstring to the format function [here](https://github.com/blendin/pwndbg/blob/ptmalloc/pwndbg/chain.py#L60). Modeled after the docstring for the enhance function.